src/open_inwoner/templates/pages/profile/categories.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'master.html' %}
+{% load i18n form_tags %}
+
+{% block content %}
+<h1 class="h1" id="title">
+    {% trans "Mijn Interessegebieden" %}
```

Comment:
Elsewhere we have the pattern <Capital non-capital>, e.g. "Mijn vragen". Also, see my question about "interessegebieden"/"onderwerpen" above.